common/model-views.h

```diff
@@ -606,14 +625,19 @@ namespace rs2
         bool draw_plane = false;
 
         bool draw_frustrum = true;
-        bool syncronize = false;
+        bool support_non_syncronized_mode = true;
+        bool syncronize = true;
```

Comment:
should be `atomic` - used inside the sensor callback

Reply:
Also - I would rename this to something that describes a state (e.g is_synchronized / synchronization_enable etc..)